### ISSUE TITLE
Discussion: greyscale

### ISSUE THREAD

**Issue by gvwilson** [merely-useful/py-rse#605]
Please vote up or down on CRC Press's proposal to print the book in greyscale. (Sample attached.)

[python-rse_grey.pdf](https://github.com/merely-useful/py-rse/files/6023340/python-rse_grey.pdf)

**Comment by joelostblom** [comment]
I would personally prefer color for the syntax highlighting (assuming there will be some applied?). I am OK with grayscale if most others want it, but think it looks less engaging, e.g.:

![image](https://user-images.githubusercontent.com/4560057/108736169-370f9400-74e6-11eb-92ca-350a4d40bbc9.png)

**Comment by lwjohnst86** [comment]
I was originally going to say, yea greyscale is fine... but didn't even check into the syntax highlighting that @joelostblom brought up... So I am going to agree with Joel about colour just for that reason. 

Also, partly related... why do some of the images look smaller? We fixed them previously...

**Comment by k8hertweck** [comment]
Yeah, we need to fix those figure sizes. Sigh.

If we do leave in greyscale, there are a few places we may need to update language-- in particular, the caption for at least one of the GitHub figures includes references to color. There may be references to color in a few other figures, too (I'm thinking of the screen caps from the packaging chapter).

This may be mitigated by explicitly stating that the figures are available online in color? It's a lackluster solution; not sure what the balance is between needing to update text vs color choice at this point.

**Comment by DamienIrving** [comment]
I agree that syntax highlighting is pretty important - it's hard to parse large sections of code without it. The price differential isn't that large, so I'd be inclined to vote for color printing.

Picking up on @lwjohnst86's comment, the sample manuscript is definitely not the latest version that we sent to them. As well as the miniature figures (which we fixed), there's also a bunch of overflow text (e.g. p 326) that we also fixed previously. 

@gvwilson Do we need to check that they're working with the correct version of the manuscript?
